api/v2/middlewares/verifyAdmin.js

```diff
@@ -10,24 +7,17 @@ import db from '../config/connection';
 */
 const verifyAdmin = async (req, res, next) => {
   try {
-    const { userId } = req.body;
-    const getUser = await db.query(findById(userId));
-    if (getUser.rowCount === 0) {
-      return res.status(404).json({
-        status: 'user could not found',
-        message: 'Invalid username or password!',
-      });
-    }
-    if (getUser.rows[0].username !== 'adminuser') {
+    if (req.userId.email !== 'adminuser@gmail.com') {
```

Comment:
I don't think this method of verifying admin is scalable. What happens when the admin changes his/her email address, you will have to alter your code then??